### PR TITLE
Feature/35521 ium capabilities

### PR DESCRIPTION
**Pull request by b12f**
Tests:

Happy paths:

- [x] Invite existing user
- [x] Invite new user by email
- [x] Add existing placeholder
- [x] Create new placeholder
- [x] Add group

Less happy paths:

- [x] No EE license: cannot add or create placeholders
- [x] Only show projects for which you have manage_members rights
- [x] Only allow to invite user by email if you have manage_users rights
- [x] Only allow to create new placeholder if you have manage_placeholder_user rights
- [x] Don't offer to invite users/groups/placeholders that are already in the project

### CODE REVIEW

**Review by oliverguenther — APPROVED** [review]
I restored the shrinkwrap file that got deleted in the PR. I also added JSdoc `@deprecated` to the legacy currentUserService methods so that IDEs get notified of that.

LGTM otherwise :+1:

**Comment by b12f** [comment]
Cool, thanks! @aleixsuau do you want to take a look at the Akita store here as well?

File: frontend/src/app/modules/common/openproject-common.module.ts [hunk 1]
```diff
@@ -124,6 +119,7 @@ export function bootstrapModule(injector:Injector) {
     DragulaModule,
     // Our own A11y module
     OpenprojectAccessibilityModule,
+    CurrentUserModule,
```
Comment (aleixsuau) [inline frontend/src/app/modules/common/openproject-common.module.ts:122]: If it is possible, I would import the CurrentUserModule directly where it is needed. This way the dependencies between modules are more clear and decoupled.

**Review by aleixsuau — COMMENTED** [review]
It is nice to have Akita in the project :)

File: frontend/src/app/modules/common/openproject-common.module.ts (hunk 1 above)
Comment (oliverguenther) [reply frontend/src/app/modules/common/openproject-common.module.ts:122]: I think it's fine to expose that globally. I'd even put it in the app module when we have one. The services should be global singulars and no components are exported. 

Basically, the only reason to have a module is to extend the sentry scope.

**Pull request merged**